src/scancode/cli.py

```diff
@@ -223,20 +232,39 @@ def reindex_licenses(ctx, param, value):
 number of files processed. Use --verbose to display file-by-file progress.
 '''
 
+formats = ('json', 'json-pp', 'html', 'html-app', 'spdx-tv', 'spdx-rdf')
+plugin_formats = {}
 
 class ScanCommand(BaseCommand):
     short_usage_help = '''
 Try 'scancode --help' for help on options and arguments.'''
 
+    options = [ op for op in scan_proper_pm.hook.add_cmdline_option() if isinstance(op, click.Option) ]
+
+    def __init__(self, name, context_settings=None, callback=None,
+                 params=None, help=None, epilog=None, short_help=None,
+                 options_metavar='[OPTIONS]', add_help_option=True):
+        BaseCommand.__init__(self, name, context_settings, callback,
+                 params, help, epilog, short_help,
+                 options_metavar, add_help_option)
+        plugins_ret_list = scan_proper_pm.hook.add_format()
+        for plugin_ret in plugins_ret_list:
```

Comment:
`plugin_ret` is the list of exactly 2 items (as I mentioned in the above comment).